src/app/components/common/dialog/downloadkey/downloadkey-dialog.component.html

```diff
@@ -1,13 +1,13 @@
 <h1 mat-dialog-title>WARNING!</h1>
 <div mat-dialog-content>
-	<p>Backup your recovery key now! If the key is lost, the data on the disks will also be lost with no hope of recovery.</p>
-	<p>>>> Click the 'Download Recovery Key' button to download your key <<<</p>
-	<p>This type of encryption is primarily targeted at users who are storing sensitive data and want the ability to remove disks from the pool and dispose of/re-use them without concern for erasure.
-	iXsystems, Inc. can not be held responsible for any lost or unrecoverable data as a consequence of using this feature.</p>
+	<p>Backup the recovery key now! If the key is lost, the data on the disks will also be lost with no hope of recovery.</p>
+	<p>>>> Click 'Download Recovery Key' to begin the download. <<<</p>
+	<p>This type of encryption is primarily targeted at users who are storing sensitive data and want the ability to remove disks from the pool and dispose of or reuse them without concern for erasure.
```

Comment:
"This type of encryption is for users storing sensitive data. Encrypted disks can be removed from the pool and reused or disposed of without being erased."